.github/workflows/go.yml

```diff
@@ -173,3 +173,35 @@ jobs:
         run: |
           move .\bin\vela .\bin\vela.exe
           .\bin\vela.exe version
+
+    image-build-check:
```

Comment:
Is this a repeat job? There is a `Registry` workflow to check building images.

Reply:
The Registry CI only works in release/master branch, I'm making it to be PR normal check.

Reply:
We could set the Registry workflow to work in all branches?

Reply:
That is a more complicate CI and contains secrets, we don't need to do that.